README.md

```diff
@@ -44,6 +44,45 @@ Alternatively using `cargo install`:
     (while sleep 5; do echo; done) | target/release/metamath-knife --timing --jobs 4 --split --repeat --trace-recalc --verify set.mm/set.mm
     # then make small changes to the beginning, end, or middle of the DB and observe how behavior changes
 
+Here is the command line help, which gives an idea of the options available:
+```
+A Metamath database verifier and processing tool
+
+USAGE:
+    metamath-knife [FLAGS] [OPTIONS] <DATABASE>
+
+FLAGS:
+        --debug                Activates debug logs, including for the grammar building and statement parsing
+        --free                 Explicitly deallocates working memory before exit
+    -g, --grammar              Checks grammar
+    -h, --help                 Prints help information
+    -O, --outline              Shows database outline
+    -p, --parse-stmt           Parses all statements according to the database's grammar
+    -t, --parse-typesetting    Parses typesetting information
+    -F, --print-formula        Dumps the formulas of this database
+    -G, --print-grammar        Prints the database's grammar
+        --dump-typesetting     Shows typesetting information
```

Comment:
`--print-...` is described as either "dumps" or "prints", and then `--dump-...` is described as "shows". Not very consistent.
Also, for consistency, `--dump-typesetting` could be abbreviated `-T`?

Reply:
Ok, so in my opinion they are all dumps. Maybe:

- `-print-formula` should become `-dump-formula`
- `-print-grammar` should become `-dump-grammar`
- The comment for `-dump-typesetting` should be "Dumps" instead of "Shows"

Reply:
Changed in last commit.